Makefile

```diff
@@ -334,6 +337,7 @@ install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	done
 
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
+	$(INSTALL_M) $(build_bindir)/embedding* $(DESTDIR)$(bindir)/
```

Comment:
In #8757 @tkelman suggests that it should be tested against an installed `julia`, hence actually installing it. I personally don't like it being in `bindir` either, but it will break if it is installed anywhere else currently. It is one of my personal crusades to get embedding to the point that embedding executables are easily relocatable, but we aren't there yet. I am actively working on it, however.